test/suite/test_bulk01.py

```diff
@@ -184,6 +184,7 @@ def test_bulk_load_order_check(self):
 
     # Test that row-store bulk-load out-of-order can succeed.
     def test_bulk_load_row_order_nocheck(self):
+        self.skipTest('Changed the error return to an assertion')
```

Comment:
This failing test checks if the return error message is expected. since we have changed it to an assert, we can skip this test

Reply:
Can you run this test to run in the non-diagnostic build? So that we have both the assert and the test.

Reply:
This test originally only allowed to be run in diagnostic build, so i dont think its supposed to be ran on non-diagnostic builds

Reply:
ok, understand.